test/db-migrations/20250415-01-client-audit-remainder.spec.js

```diff
@@ -0,0 +1,82 @@
+const { hash, randomBytes } = require('node:crypto');
+
+const { // eslint-disable-line object-curly-newline
+  assertTableSchema,
+  assertTableContents,
+  describeMigration,
+} = require('./utils'); // eslint-disable-line object-curly-newline
+
+describeMigration('20250415-01-client-audit-remainder', ({ runMigrationBeingTested }) => {
+  const aBlobWith = props => {
+    const randomContent = randomBytes(100);
+    const md5 = hash('md5',  randomContent); // eslint-disable-line no-multi-spaces
+    const sha = hash('sha1', randomContent);
+    return { md5, sha, ...props };
+  };
+  const aBlob = () => aBlobWith({});
```

Comment:
Since these functions are also used elsewhere, how about moving them into a utility file? I'm sure we'll need them again in future migrations.